Unified the formatting and added rail road diagrams (#39)
UPPAALModelChecker/docs.uppaal.org: content/language-reference/requirements-specification/_index.md

`control: A[] safe`

> : compute a strategy where `safe` state predicate is always true no matter what the oponent/environment chooses to do. The strategy is *permissive* in a sense that for a given state the strategy may propose multiple actions for the player/controller. Such permissive strategy can be thought of as a union of all strategies satisfying the predicate, therefore it does not have any notion of progress and may include infinite loops.

Reviewer: ```suggestion
: compute a strategy where `safe` state predicate is always true no matter what the opponent/environment chooses to do. The strategy is *permissive* in a sense that for a given state the strategy may propose multiple actions for the player/controller. Such permissive strategy can be thought of as a union of all strategies satisfying the predicate, therefore it does not have any notion of progress and may include infinite loops.
```

`E<> control: A[] safe`

> : computes a collaborative strategy, i.e. finds a reachable state from where a safe strategy is found: `safe` state predicate is maintained to always remain true no matter what the oponent/environment chooses to do.

Reviewer: This description didn't really explain the query to me, i've tried to change the sentence to highlight what i experienced/suprised me while testing this query type
```suggestion
: Checks if opponent and player can work together and reach one of the winning states of a given control strategy, note that the state predicate `safe` must always be true before reaching a winning state and is guaranteed to remain true after reaching winning state.

> `{ x<=5, Gate.Idle, Gate.Occ }: A[] safe`

Reviewer: This line doesn't match the CFG given above.
By looking at the PartialControlSpecifier I expect the syntax:
`{ x<=5, Gate.Idle, Gate.Occ } control: A[] safe`

Author: Good catch!

> `{ x<=5, Gate.Idle, Gate.Occ }: A[] safe`

Reviewer: This query type does not work in Uppaal 5 it throws a segmentation fault when trying

> : computes a safe strategy by observing only the state of the specified predicate expressions.

Reviewer: ```suggestion
: computes a safety strategy where the player can only observe the state of the specified predicate expressions.
```

`A[ safe U goal ]`

> : computes a safe strategy but only up until the `goal` is reached.

Reviewer: ```suggestion
: computes a safety strategy but only up until the `goal` is reached.
```

`A[ safe W goal ]`

> : (weakly until the `goal`) either a safe strategy is found or a safe strategy holds until the goal is reached.

Reviewer: ```suggestion
: (weakly until the `goal`) either a safety strategy is found or a safety strategy holds until the goal is reached.
```
